src/Frontend/Modules/Profiles/Actions/Register.php

```diff
@@ -118,10 +118,7 @@ private function validateForm(): void
 
                 // values
                 $values['email'] = $txtEmail->getValue();
-                $values['password'] = FrontendProfilesModel::getEncryptedString(
-                    $txtPassword->getValue(),
-                    $settings['salt']
```

Comment:
In the profile module it is also used for reset password key and activation key, so can't be removed completely.